index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
-concrete examples. It is divided into two parts. On the one hand, there is the
-structure and metadata specification which defines how the AIP is conceptually
-organized by means of a folder hierarchy in a file system and a set of
-metadata standards. And on the other hand, there is the physical container
-specification which defines the bit-level manifestation of the transferable
-entity.
+A version of an AIPs is created due to some change in the content or metadata, or 
+a policy for a particular collection:
 
-[^2]: The requirements terminology is based upon RFC2119, "Key words for use in
-      RFCs to indicate requirement levels", RFC 2119, S. Bradner, March 1997.
-      Available at: http://www.ietf.org/rfc/rfc2119.txt
+- Extending, correcting or removing package metadata
+- Adding, replacing or removing content items 
 
-<a name="csip"></a>
 
-## Overview about the CSIP
+## Segmentation of the AIP <a name="structdiv"></a>
 
-In the following, we will briefly describe the structure of an IP as defined
-by the CSIP. Against this background, the AIP format will be introduced as a
-container format which allows managing the life-cycle of an E-ARK IP that
-starts with the ingest of an SIP.
+From the point of view of preserving the integrity of the AIP, the ideal case 
+isthat the logical AIP is packaged as one single physical container, because 
+all themetadata and content required to interprete the information package is 
+available inone single entity. In reality, however, this is not always possible 
+because the size of the physical container can become very large.
 
-As already mentioned in section [4.1](#reps), the CSIP relies on the concept of
-"representations". Figure [1](#fig1) gives an example of the structure of an IP
-with two representations.
+For this reason, the AIP format describes how to partition the AIP and keep 
+representations or representation parts in separate physical container files 
+(see section [5.1](#compdiv)). Even though this puts the integrity of the AIP 
+at risk - because in case of disaster recovery the physical container does not 
+represent the complete intellectual entity and dependencies to another (lost) 
+physical container can potentially make it impossible to interpret, understand, 
+or render the content - it is a necessary measure if the amount of data exceeds 
+the capacity limitation of long-term storage media.
 
-<a name="fig1"></a>
-![Information Package structure](figs/visio/fig_1_ip_structure.svg "Structure of an IP compliant with the E-ARK Common Specification for IPs structure")
+*Definition:* *Segmentation* is a physical manifestation of a logical AIP where a set of 
+physical container files contains parts of the logical AIP. Each segment of the logical AIP 
+is a packaged as a TAR or ZIP file and contains its own structural metadata.
 
-**Figure 1:**
-Structure of an IP.
+When the AIP is segmented, the structure of the segmentation needs to be recorded in the
```

Comment:
structure of the segmentation

This might benefit from a rewording, struggling to think of something appropriate ATM.

Reply:
This is how I reworded it: "It is mandatory to document the segmentation of an AIP in the structural metadata."

Does that work?